types/wlr_output_management_v1.c

```diff
@@ -819,6 +819,25 @@ static bool manager_update_head(struct wlr_output_manager_v1 *manager,
 		state |= HEAD_STATE_SCALE;
 	}
 
+	// When a mode was added to wlr_output.modes we need to add the new mode to the wlr_output_head
```

Comment:
style nit: you should wrap this comment to 80 columns, also "if" sounds better than "when" here.